openapi.yaml

```diff
@@ -659,7 +659,7 @@ components:
           description: |
             A well defined set of severity strings guaranteed to be present.
           type: string
-          enum: [Unknown, Negligible, Low, Medium, High, Critical, Defcon1]
```

Comment:
@syed @alecmerdler 

Will Quay need any updates in it's API validator since the openapi spec is changing now?

Reply:
I think I removed it from Quay as well https://github.com/quay/quay/pull/664/files#diff-de511d8a0ffe2df62730e731ccb49271e73c4f276e26bd5a8e4a757e46beb26f

Reply:
Cool I think that would do it.